source/slides/10_cfg_mgt.rst

```diff
@@ -2,3 +2,74 @@
 
 Configuration Management History & Basics
 =========================================
+
+
+DevOps workflow in an agency environment
+========================================
+
+Squishymedia
+
+.. image:: /_static/squishy.png
```

Comment:
Because of how our repo is setup, can you please change this to ``../_static/squishy.png``?